docs/OTCamera/gettingstarted/installation.md

```diff
@@ -1,194 +1,46 @@
-# Installation
+# Overview
 
 No matter which version you want to install, you will always use a Raspberry Pi as a base.
-This guide describes the basic steps you need to perform for each version.
 
 You will need:
 
-* Raspberry Pi 2B/3B(+)/4/Zero W and power supply
+* Raspberry Pi 2B / 3B(+) / 4 / Zero W / Zero 2W and power supply
+* We recommend a Zero W or Zero 2W because they draw substantially less power.
 * Micro SD card (a High Endurance version is recommended)
 * SD Card Reader
 * [Raspberry Pi Imager](https://www.raspberrypi.org/software/).
 
 !!! warning
 
     Even though we have completed the guide ourselves, every PC is different and Raspberry OS can change as well.
-    There are steps described with which you can break your Windows on the PC.
+    There are steps described with which you can break your Windows, macOS or Linux on your PC.
     You should know what you are doing.
     We are not responsible for any damage that may occur.
 
-## Prepare the SD Card
+We provide a simple quick installer which will do most of the work for you.
 
-Download and install the [Raspberry Pi Imager](https://www.raspberrypi.org/software/), insert the SD-Card and start the Imager.
+!!! info
+    If you like, you can also setup the Raspberry Pi and install OTCamera manually.
+    The necessary steps are described on the following pages.
```

Comment:
Didn't get it.

Reply:
Using the quick installer, isnt the only manual step bringing the raspberry os image to the sd card? Then a link to this specific page would be helpful (instead of scrolling through all pages describing the manual steps)

Reply:
k. there already is a link to "prepare sd card". I also added a backlink in "prepare sd-card"